test/suites/alpha/integration/extended_resources_test.go

```diff
@@ -488,3 +606,181 @@ func ExpectHabanaDevicePluginCreated() {
 		},
 	})
 }
+
+func ExpectNeuronDevicePluginCreated() {
+	GinkgoHelper()
+	env.ExpectCreated(&rbacv1.ClusterRole{
+		ObjectMeta: metav1.ObjectMeta{
```

Comment:
```suggestion
		ObjectMeta: test.ObjectMeta(metav1.ObjectMeta{
```

Should we use `test.ObjectMeta` here to make this discoverable by the cleanup tool in the `AfterEach`?

Reply:
I don't believe thats required since it's handled in `ExpectCreated`: https://github.com/aws/karpenter/blob/3be36eef51a056130d7c6ef03963c6e2934ee8c5/test/pkg/environment/common/expectations.go#L54